src/components/Onboarding.jsx

```diff
@@ -0,0 +1,144 @@
+import * as rb from 'react-bootstrap'
+import React, { useState } from 'react'
+import Sprite from './Sprite'
+import { useSettingsDispatch } from '../context/SettingsContext'
+
+export default function Onboarding() {
+  const [step, setStep] = useState(0)
+  const settingsDispatch = useSettingsDispatch()
+
+  if (step === 0) {
+    return (
+      <div class="mx-auto my-auto py-5">
+        <div class="d-flex justify-content-center">
+          <Sprite symbol="logo" width="200" height="200" className="mb-4" />
+        </div>
+        <h1 class="text-center">JoinMarket</h1>
+        <p class="text-center fs-4 text-secondary mb-5">Top-notch privacy for your bitcoin.</p>
+        <div class="d-flex justify-content-center mb-2">
+          <rb.Button size="lg" className="col-10" variant="dark" onClick={() => setStep(1)}>
+            Get started
+          </rb.Button>
+        </div>
+        <div class="d-flex justify-content-center">
+          <rb.Button variant="dark" onClick={() => settingsDispatch({ showOnboarding: false })}>
```

Comment:
Let's make that the same size as the other button but `outline-dark` instead of `dark`. What do you think?

<img width="368" alt="Screenshot 2022-02-08 at 10 05 04" src="https://user-images.githubusercontent.com/10026790/152953763-c16a691c-3d1c-4355-85b4-3dd27b5c72cf.png">

```suggestion
          <rb.Button
            size="lg"
            className="col-10"
            variant="outline-dark"
            onClick={() => settingsDispatch({ showOnboarding: false })}
          >
```

Reply:
Yes that looks much better!